### ISSUE TITLE
jspm install --save and unbundle and clean remove fields from babelOptions

### ISSUE THREAD

**Issue by qubyte** [jspm/jspm-cli#630]
I've noticed that the following config:

```javascript
"babelOptions": {
  "optional": [
    "runtime"
  ],
  "experimental": true
}
```

gets reduced to

```javascript
"babelOptions": {
  "optional": [
    "runtime"
  ]
}
```

each time I run `jspm install --save ...`, `jspm unbundle` or `jspm clean`. This then has to be re-added each time, which is not desirable.

**Comment by guybedford** [comment]
Thanks I've added a fix for the next patch release.

**Comment by qubyte** [comment]
:+1: Thanks

**Comment by guybedford** [comment]
Fixed in 0.15.1
